ward/cparser.lua

```diff
@@ -765,6 +765,7 @@ local grammar = pattern {
   float_type =
     keyword "float" * value(type_float) +
     keyword "double" * value(type_float) +
+    keyword "_Float128" * value(type_float) +
```

Comment:
Going beyond this PR, I also notice that `_Bool` is referred to as a GCC extension, but it really is part of C99, so that comment should also be fixed (not by this PR, just saying it while I notice it). But this PR could perhaps add a comment explaining that `_Float128` is an ISO extension?